src/components/users/users.test.ts

```diff
@@ -429,6 +429,32 @@ describe('users test suite', async () => {
     expect(response.body).toContain('Updated a team member');
   });
 
+  it('should update the user, remove BillingManager role and show success - User Edit', async () => {
+    const scope = nock(ctx.app.cloudFoundryAPI)
+    // tslint:disable:max-line-length
+    .delete('/v2/organizations/3deb9f04-b449-4f94-b3dd-c73cefe5b275/billing_managers/uaa-user-changeperms-123456?recursive=true').reply(200, `{}`)
```

Comment:
These URIs are pretty hefty, and scattered through this file, with repeated GUIDs.

We could reduce the mental and maintenance burden by extracting the repeated GUIDs in to variables and using [template strings](https://www.typescriptlang.org/docs/handbook/basic-types.html#string) to cleanly build the URIs.